### PR TITLE
Adds rule for impossible travel (Okta)

### PR DESCRIPTION
**Pull request by brachera**

### CODE REVIEW

File: rules/cloud/okta/okta_impossible_travel_detected.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2022/09/26
+references:
+    - https://help.okta.com/en-us/Content/Topics/Security/behavior-detection/about-behavior-detection.htm
+    - https://help.okta.com/en-us/Content/Topics/Security/behavior-detection/logs-behavior-detection.htm
+logsource:
+  product: okta
+  service: okta
+detection:
+    selection:
+        okta.event_type:
```
Comment (frack113) [inline rules/cloud/okta/okta_impossible_travel_detected.yml:15]: HI,
In other rule get `eventtype` and `displaymessage` for the field name.

Comment (Neo23x0) [reply rules/cloud/okta/okta_impossible_travel_detected.yml:15]: I don't understand that comment

Comment (frack113) [reply rules/cloud/okta/okta_impossible_travel_detected.yml:15]: I don't have okta but in the other rule find `eventtype` and `displaymessage` .
missing the question: 
Are you sure of the name?

Comment (nasbench) [reply rules/cloud/okta/okta_impossible_travel_detected.yml:15]: Ping @brachera

**Comment by nasbench** [comment]
Closing this due to inactivity.

**Pull request closed without merging**